### ISSUE TITLE
Playback error keeps occurring, cant find a way round

### ISSUE THREAD

**Issue by MadMacMig** [xteve-project/xTeVe#358]
**Describe the bug**
I've been having issues with my live tv system blanking out after around half an hour and the screen would just go black, I would then have to leave the channel and go to another channel then go back to continue watching it.
So nor recently, every time I go to start a live tv feed, it starts spinning and then shows up an error, I've tried all sorts, refreshing the m3u file, restarting my server, removing from plex and reading to no success.

**Server (please complete the following information):**
- OS: Windows 10
- Arch: 64bit
- xTeVe Version: 2.2.0 Build: 0200

**Screenshots**
<img width="1680" alt="Screenshot 2023-03-09 at 21 58 21" src="https://user-images.githubusercontent.com/61500100/224169637-cad68984-e0be-4489-a309-58ee524a3b49.png">

**Desktop (please complete the following information):**
 - OS: macOS Ventura
 - Browser Safari
 - Version Latest

**Comment by MCTyler** [comment]
IPTV is unreliable. Are you saying it plays without issue without xTeVe?

**Comment by MadMacMig** [comment]
Yes it does, sorry for the late reply, it started working when they sent me a new file, then just stopped, it's really getting on my nerves. I asked for a new file so they sent me some to try, I tried another and it was working fine until literally today, it stopped working ONCE AGAIN, after working flawlessly for around 3 days.

**Comment by MadMacMig** [comment]
But when I play it through an IPTV player such as GSE IPTV for mac, it works no problem

**Comment by alcos-technologies** [comment]
I have the same issue here but not for all streams.
I can get LCN & TVA but not Radio-Canada, RDI or Télé-Québec
When I get my M3U url from xteve and I put it in VLC, i can stream all of them but not inside Plex
![image](https://github.com/xteve-project/xTeVe/assets/149615881/527e0c1c-f2e5-4458-94a6-ec4c285e6f7a)
